og.module

```diff
@@ -160,7 +161,7 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
     $handler = Og::getSelectionHandler($field_definition);
 
     if ($handler->getReferenceableEntities()) {
-      return Accessresult::neutral();
+      return AccessResult::neutral();
```

Comment:
PHP classnames are not case sensitive but file systems might be so this is an interesting bug which might not trigger ever... but it's better be fixed.